.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+
+updates:
+  - package-ecosystem: "bundler"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    allow:
+      - dependency-type: "all"
+    open-pull-requests-limit: 20
```

Comment:
Propose to remove this option, since the default value 5, so I think will be enough for now.